networking/external_dns_operator/nw-control-dns-records-public-hosted-zone-aws.adoc

```diff
@@ -0,0 +1,63 @@
+[id="nw-control-dns-records-public-hosted-zone-aws_{context}"]
+= Controlling DNS records on an public hosted zone for AWS by using Red Hat External DNS Operator
```

Comment:
Creating DNS records on an public hosted zone for AWS by using Red Hat External DNS Operator